.gitignore

```diff
@@ -1,6 +1,14 @@
 # Cargo
 /target
 
+# Generated documentation
+developer_docs/
+docs/book/
+docs/build/
+docs/dist/
+public/
+site/
```

Comment:
Why is this necessary? I thought documention is generated within `/target`, which is already ignored.